src/svgSprite.ts

```diff
@@ -293,6 +293,12 @@ export const svgSprite = (): string => `
       fill=${variables.palette.primary50 || 'currentColor'} fillOpacity="0.87" />
     </symbol>
 
+    <symbol viewBox="0 0 24 24" id="svg-dropdown-down-outlined">
+      <path d="M16.9997 9.1697C16.8123 8.98345 16.5589 8.87891 16.2947 8.87891C16.0305 8.87891 15.7771 8.98345 15.5897 9.1697L11.9997 12.7097L8.4597 9.1697C8.27234 8.98345 8.01889 8.87891 7.7547 8.87891C7.49052 8.87891 7.23707 8.98345 7.0497 9.1697C6.95598 9.26266 6.88158 9.37326 6.83081 9.49512C6.78004 9.61698 6.75391 9.74769 6.75391 9.8797C6.75391 10.0117 6.78004 10.1424 6.83081 10.2643C6.88158 10.3861 6.95598 10.4967 7.0497 10.5897L11.2897 14.8297C11.3827 14.9234 11.4933 14.9978 11.6151 15.0486C11.737 15.0994 11.8677 15.1255 11.9997 15.1255C12.1317 15.1255 12.2624 15.0994 12.3843 15.0486C12.5061 14.9978 12.6167 14.9234 12.7097 14.8297L16.9997 10.5897C17.0934 10.4967 17.1678 10.3861 17.2186 10.2643C17.2694 10.1424 17.2955 10.0117 17.2955 9.8797C17.2955 9.74769 17.2694 9.61698 17.2186 9.49512C17.1678 9.37326 17.0934 9.26266 16.9997 9.1697Z"
+      fill=${variables.palette.on_surface_variant || 'currentColor'} />
```

Comment:
I realize this is a pattern in the rest of this file, but I feel like we should always be using `currentColor` in these sprites and leave it up to the consumers of the `<Svg />` component to decide the fill, rather than having the sprite have a default color that would need to be overwritten

Reply:
+1

Reply:
Sure, I think the result would be the same so I'm happy to break the pattern here